### DISCUSSION TITLE
青模网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#140] [模板讨论]
青模网 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
签到结果：【SLIDER】
当前下载币为：【50】，余额为：【0】，卡券为：【0】

签到显示成功，也返回了数据，但是很长时间停留在50，没变了。
手工登录网页，打开签到页，没有成功（灰色），手动点击，显示成功（彩色）。

**Reply by wjf0214** [reply]
SLIDER,你这遇到滑块验证了。

**Comment by gurenn** [comment]
哈？青模网的签到脚本有了吗？怎么更新公共库没看到？

**Reply by wjf0214** [reply]
一直都有。只是cookie有效期就7天。
